journal/updated/07-chapter-6.adoc

```diff
@@ -606,16 +609,16 @@ in order that a few living stones may be supporters of a holy building.
 O, my dears, look to it, and remember what station you are in.
 In the love of the Gospel I beseech you to take heed how you walk,
 and that you so demean yourselves as to be ensamples to the flock of Christ,
-and feeders of his lambs--way-marks unto all, both within and without.
-We are watched, and we must watch also, even against every appearance of evil;
+and feeders of His lambs--way-marks unto all, both within and without.
+We are watched, and we must watch also, even against every appearance of evil,
 and not allow our own wills to prevail,
 but let our dispositions be brought into true subjection,
 that we may experience the truth of the lion and the lamb lying down together,
 and of the weaned child playing at the hole of the asp,
 and the sucking child laying its hand upon the cockatrice`'s den,
 and that nothing can hurt or destroy upon God`'s holy mountain.
 I had no thought of penning these things when I sat down,
-but merely to give you an account of my welfare, as through mercy I am bravely,
+but merely to give you an account of my welfare; through mercy I am doing bravely,
```

Comment:
brave or bravely is another word that I sometimes change. Here it actually just means, I am doing **well**,